Bottom: d901bff8f644702c559d4ff810b2e8d765d3b3b6
Top:    d064a01efcd5ab68de48feaff6725d4ff4bd0cc0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-01 23:43:00 +0000

hid/gcode: Add #include for locale.h required for setlocale()


---

diff --git a/src/find.c b/src/find.c
index 4c897ee..16015dc 100644
--- a/src/find.c
+++ b/src/find.c
@@ -159,7 +159,7 @@ static bool IsRatPointOnLineEnd (PointType *, LineType *);
 static bool ArcArcIntersect (ArcType *, ArcType *);
 static bool PrepareNextLoop (FILE *);
 static void DrawNewConnections (void);
-/* XXX */ static bool SetThing (int, void *, void *, void *);
+///* XXX */ static bool SetThing (int, void *, void *, void *);
 static bool IsArcInPolygon (ArcType *, PolygonType *);
 static bool IsLineInPolygon (LineType *, PolygonType *);
 static bool IsPolygonInPolygon (PolygonType *, PolygonType *);
diff --git a/src/hid/gcode/gcode.c b/src/hid/gcode/gcode.c
index 9a0894f..208d02d 100644
--- a/src/hid/gcode/gcode.c
+++ b/src/hid/gcode/gcode.c
@@ -45,6 +45,10 @@
 
 #include <time.h>
 
+#ifdef HAVE_LOCALE_H
+#include <locale.h>
+#endif
+
 #include "global.h"
 #include "error.h" /* Message() */
 #include "data.h"
